Apps/WebClient/src/ClientApp/src/components/report/covid19.vue

```diff
@@ -151,7 +155,12 @@ export default class COVID19ReportComponent extends Vue {
                         {{ item.location }}
                     </b-col>
                     <b-col data-testid="covid19ItemResult" class="my-auto">
-                        {{ item.labResults[0].labResultOutcome }}
```

Comment:
not sure this is necessary

Reply:
@FuriousLlama can you elaborate which one is not necessary and why?

Reply:
The additional check to only show if it is final for report is not outlined on the story. If the lab outcome shows currently, then there is no change on the reports side

Reply:
@FuriousLlama I think the story 10222 is asking for showing it only for `validate "labresultoutcome" is showing for "Amended", "Final" or "Corrected" status `
@ninosamson can you confirm this?

Reply:
So we should be validating that we show result outcome if Amended, Corrected or Final. 

In the report, it looks like we're not showing the outcome on the amended Dec 3 test.

Reply:
Also, the Amended test is on Sam Testfive - we should validate that the result and report show on that user.

Reply:
On timeline, dependent, and export, we have to show the result (labresultoutcome) and report (PDF available) if status = Final, Amended, or Corrected

Reply:
Thanks @ninosamson. Does this apply to the Covid-19 report as well? 
![image](https://user-images.githubusercontent.com/48332333/109911689-5d100380-7c5f-11eb-8158-da7ec9bd619e.png)

Reply:
Yep, test on the 14th is Amended, so we should show the result. 

Is the "labresultoutcome" notset?

Reply:
It would normally have a "labresultoutcome"

Reply:
So the expectation is that for this user,
On jun 14 and dev 5, there is a labresultoutcome (result) showing.

Reply:
On Dependent Sam Testfive, the nov 10 viha test should show result and report

Reply:
thanks @ninosamson , the screenshot was taken for my local test data :-) So we should hide and show the data in the Result column based on the testStatus as in the user story.

Reply:
Please correct me if I am wrong. I think the 'validate' part of the user acceptance criteria means "make sure that the data shows for these statuses"  and not "add validation so that it only shows for these statuses"
So in summary what I understand is that it needs to show for those statuses for sure. Other statuses are also welcomed. That is in contrast to only showing when the status is final (like the other changes)

Is that right @ninosamson?